### DISCUSSION TITLE
Arch Linux packages

### DISCUSSION THREAD

**Discussion by moonlit-tune** [markqvist/Reticulum#91] [General]
Just a quick announcement that I've packaged [rns](https://aur.archlinux.org/packages/python-rns), [lxmf](https://aur.archlinux.org/packages/python-lxmf), [nomadnet](https://aur.archlinux.org/packages/nomadnet), [rnodeconf](https://aur.archlinux.org/packages/rnodeconf), and [loramon](https://aur.archlinux.org/packages/loramon) for Arch Linux. They should all be available from the AUR as of now.

For the time being I've held off on packaging rnspure because I'm not aware of any cases in which one would have access to the AUR but be unable to use OpenSSL (Arch Linux ARM packages OpenSSL for armv7h and aarch64).

Hope this is welcome/helpful! Please let me know if you have any questions or feedback.

**Comment by markqvist** [comment]
This is insanely great! Thanks so much for doing this, I will update the docs for installation this way too on Arch.

How does updates work? Are you manually packaging new versions for AUR when I make a release here on github or PyPi?

**Reply by moonlit-tune** [reply]
Thanks likewise; really excited about the project!

Yes, as of right now packaging is manual. I track rss feeds from PyPi, and barring any structural changes, the process is simply "bump version numbers and checksums in the PKGBUILD file, regenerate the .SRCINFO file, and push to the AUR." I don't see any reason why this couldn't be automated as part of your release process if that's something you're interested in, but I'm fine handling it like this as well.

**Reply by markqvist** [reply]
Ah great, then it is not too involved for you :) Let me know if there is anything I need do to make it easier down the line!

**Comment by fotografm** [comment]
I installed reticulum-meshchat-bin on EndeavourOS and it cannot access my Heltec v3 Rnode via usb cable. This works fine on Mint 22 after I added my user to the dialout group. Any tip on how I can get this to work on EndeavourOS ?

**Reply by markqvist** [reply]
The serial port access group might be called something different on that OS. Could be `uucp`, `serial` or `dialout` as far as I remember. Probably something else as well.

**Reply by liamcottle** [reply]
For future reference if anyone else comes across this:
https://github.com/liamcottle/reticulum-meshchat/issues/37

**Comment by fotografm** [comment]
Hi Mark
Thank you for your quick answer. The only groups I seem to have on
EndeavourOS are sys, wheel, rfkill and user.
Liam Cottle suggested in a Github thread (I think he sent the link to you
by email) that I try adding a udev rule but no success so far. Am still
experimenting.
Cheers
Simon.
